kurtosis-devnet/pkg/kurtosis/sources/spec/spec.go

```diff
@@ -46,14 +46,14 @@ type ChainConfig struct {
 }
 
 // InteropConfig represents the interop section in the YAML
-type InteropConfig struct {
+type SuperchainConfig struct {
 	Enabled bool `yaml:"enabled"`
```

Comment:
Are we not including `superchain > L2` mapping here?

Reply:
This thing is really parsing the least  amount of network_params.yaml possible to figure out high level features (like interop)